### PR TITLE
Redirect without message from `/`.

### PR DESCRIPTION
**Pull request by NoLogo**
@incuna/backend Please review. 
(I'm open to ideas if anyone has a nicer way of doing this)

### CODE REVIEW

**Comment by NoLogo** [comment]
Updated per @adam-thomas's suggestion IRL.

File: tests/test_permission.py [hunk 1]
```diff
@@ -108,8 +109,12 @@ def test_deny_access_condition_allow(self):
 
     def test_access_denied_message(self):
         """Assert the message returned by get_access_denied_message."""
+        request = self.create_request(url='/testing/')
         expected_message = 'You must be logged in to view this page.'
-        self.assertEqual(expected_message, self.middleware.get_access_denied_message())
+        self.assertEqual(
+            expected_message,
+            self.middleware.get_access_denied_message(request)
```
Comment (adam-thomas) [inline tests/test_permission.py]: You can just pass `request=None` here (which is also a stronger test).

File: tests/test_permission.py [hunk 2]
```diff
@@ -81,11 +81,12 @@ def test_configured_unauthorised_redirect_url(self):
...
         self.assertEqual(url, self.middleware.get_unauthorised_redirect_url(request))
 
-    def test_default_access_denied_message(self):
+    def test_access_denied_message(self):
         """
         Assert the default implementation of get_access_denied_message returns ''.
         """
-        self.assertEqual('', self.middleware.get_access_denied_message())
+        request = self.create_request()
+        self.assertEqual('', self.middleware.get_access_denied_message(request))
```
Comment (adam-thomas) [inline tests/test_permission.py]: You can pass `request=None` here as well :)

File: setup.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='incuna-auth',
-    version='3.0.1',
+    version='3.0.2',
```
Comment (adam-thomas) [inline setup.py]: `v4.0.0`

File: changelog.rst [hunk 4]
```diff
@@ -5,6 +5,10 @@ Upcoming
 --------
 * Removed use of deprecated `patterns` function from `urls.py`.
 
+v3.0.2
```
Comment (adam-thomas) [inline changelog.rst]: `v4.0.0`

**Comment by adam-thomas** [comment]
LGTM aside from the version number and some test tweaks. :)

**Comment by NoLogo** [comment]
Updated

**Comment by adam-thomas** [comment]
:+1:

**Comment by NoLogo** [comment]
Ta

**Pull request merged**